deepgram/__init__.py

```diff
@@ -12,6 +13,11 @@
 from .errors import DeepgramSetupError, DeepgramApiError
 
 
+def api_key_is_valid(api_key: str) -> bool:
+    pattern = r"^[a-z0-9]{40}$"
```

Comment:
I think it is only `a-f`, not `a-z` - is that too finicky? I think fine as-is, it will do the trick as it's an odd edge case for a user to provide a 40-char false API key.

Reply:
It's easy enough to fix, thanks!